Realm/RLMObject_Private.h

```diff
@@ -61,6 +61,8 @@ FOUNDATION_EXTERN BOOL RLMObjectBaseAreEqual(RLMObjectBase *o1, RLMObjectBase *o
 // Get ObjectUil class for objc or swift
 FOUNDATION_EXTERN Class RLMObjectUtilClass(BOOL isSwift);
 
+FOUNDATION_EXTERN const NSUInteger RLMDescriptionMaxDepth;
```

Comment:
According to https://twitter.com/jckarter/status/583725475777748992, using `static const` declarations from Swift doesn't work (and is a known bug).